build-tools/create-packs/Directory.Build.props

```
@@ -0,0 +1,12 @@
+<Project> 
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
```

I think you could put this instead and it would be clearer:

```suggestion
  <Import Project="..\..\Directory.Build.props'))" />
```

`<Import/>` should look relative to the current file.

---

I was following the recommendation here: https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019#use-case-multi-level-merging

But I agree this is probably more complex than needed, since we know which "outer" file we want to import.